src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -48,11 +52,11 @@ For more about these tools, see the [new user management docs](/docs/accounts/ac
 
 The migration wizard is meant to have all the context you need to complete the migration process but here are some tips for potential areas of confusion:
 
-* Guided setup vs automatic setup: Simple organizations with few users and few accounts, and who have little need for their users to have separate roles or separate account access, may decide to choose the automatic setup. If you require some customization of user access, or if your New Relic users use SAML SSO to sign in or are managed via an identity service provider (like Azure AD or Okta), you should use the guided setup. 
+* Guided setup vs. automatic setup: Simple organizations with few users and few accounts, and who have little need for their users to have separate roles or separate account access, may decide to choose the automatic setup. If you require some customization of user access, or if your New Relic users use SAML SSO to sign in or are managed via an identity service provider (like Azure AD or Okta), you should use the guided setup. 
```

Comment:
This is all great info! But for me, each bullet has a lot of information that is complex to understand. Maybe a followup Jira to consider restructuring all these bullets another way?